app/components/contacts/AllContacts.js

```diff
@@ -69,7 +70,7 @@ class AllContacts extends Component<Props, State> {
         </SearchRow>
         <ListsWrapper isModalMode={isModalMode}>
           <div>
-            {lastUsedAddresses && lastUsedAddresses.length && (
+            {lastUsedAddresses && lastUsedAddresses.length ? (
               <ContactsList
```

Comment:
ContactsList expects to get objects of Contact type and not Tx type

Reply:
ok